rocAL/rocAL/include/pipeline/tensor.h

```diff
@@ -81,9 +81,6 @@ class TensorInfo {
     TensorInfo(std::vector<size_t> dims, RocalMemType mem_type,
                     RocalTensorDataType data_type);
 
```

Comment:
What is the reason we have removed copy constructor and destrcutor?

Reply:
Because we are using a shared pointer and that will handle the copies